backend/api/Areas/Admin/Controllers/AccessRequestController.cs

```diff
@@ -40,25 +40,35 @@ public class AccessRequestController : Controller
         #endregion
 
         #region Endpoints
+
         /// <summary>
         /// Get a list of access requests
         /// </summary>
         /// <param name="page"></param>
         /// <param name="quantity"></param>
         /// <param name="sort"></param>
+        /// <param name="searchText"></param>
+        /// <param name="role"></param>
+        /// <param name="agency"></param>
         /// <param name="status"></param>
         /// <returns></returns>
         [HttpGet("")]
         [Produces("application/json")]
         [ProducesResponseType(typeof(PModel.PageModel<Model.AccessRequestModel>), 200)]
         [ProducesResponseType(typeof(Api.Models.ErrorResponseModel), 400)]
-        [SwaggerOperation(Tags = new[] { "admin-access-requests" })]
-        public IActionResult GetPage(int page = 1, int quantity = 10, string sort = null, Entity.AccessRequestStatus? status = Entity.AccessRequestStatus.OnHold)
+        [SwaggerOperation(Tags = new[] {"admin-access-requests"})]
+        public IActionResult GetPage(int page = 1, int quantity = 10, string sort = null,
+            string searchText = null, string role = null, string agency = null,
+            Entity.AccessRequestStatus? status = Entity.AccessRequestStatus.OnHold)
         {
             if (page < 1) page = 1;
             if (quantity < 1) quantity = 1;
             if (quantity > 20) quantity = 20;
-            var result = _pimsAdminService.User.GetAccessRequests(page, quantity, sort, status);
+            if (status == null) status = Entity.AccessRequestStatus.OnHold;
```

Comment:
Out of curiosity is this even required as you have a default value in the arguments.  Is this only required because the requests passes an empty param (i.e. `?status=&page=1`)?

Reply:
Not necessary, I changed the endpoint param to be not nullable